.github/workflows/test.yml

```diff
@@ -31,11 +31,27 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
+      - name: Cache cargo registry
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/registry
+          key: ${{ runner.os }}-${{ steps.rustup.outputs.rustc_hash }}-registry-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ steps.rustup.outputs.rustc_hash }}-registry-
+      - name: Cache cargo index
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/git
+          key: ${{ runner.os }}-${{ steps.rustup.outputs.rustc_hash }}-index-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
```

Comment:
With this fallback the cache will build up with artifacts from old dependencies.

Reply:
I'll open a PR to revert this bit later.